camera-viewer/main.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+const inputIPaddress = document.getElementById("ip-address")
```

Comment:
-  is it the correct camel case of `Input, Ip, Addrress `
- how about the `inputIpAdrress` 

## Naming: 
- Is the `inputIpAdrress` a good name? 

The value of `inputIpAdrress` variable is a DOM element